<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBConfigDialog</class>
 <widget class="QDialog" name="DBConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>配置数据库</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resource/image.qrc">
    <normaloff>:/img/icon.png</normaloff>:/img/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLineEdit" name="lineEditUserName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 80, 80);
border-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);
border:2px;
border-radius:5px;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDBConfig">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>240</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>配置数据库</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditUrl">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 80, 80);
border-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);
border:2px;
border-radius:5px;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Url</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: qlineargradient(spread:repeat, x1:0, y1:1, x2:1, y2:0, stop:0.272727 rgba(85, 85, 85, 255), stop:1 rgba(120, 120, 120, 255));
border-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);
border:2px;
border-radius:5px;
}
QPushButton:hover {
   background-color: qlineargradient(spread:repeat, x1:0, y1:1, x2:1, y2:0, stop:0.272727 rgba(120, 120, 120, 255), stop:1 rgba(85, 85, 85, 255));
}
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: rgb(120, 120, 120);
}</string>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 80, 80);
border-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);
border:2px;
border-radius:5px;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDB">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 80, 80);
border-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);
border:2px;
border-radius:5px;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>数据库</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditUrl</tabstop>
  <tabstop>lineEditUserName</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>lineEditDB</tabstop>
 </tabstops>
 <resources>
  <include location="../Resource/image.qrc"/>
 </resources>
 <connections/>
</ui>
